ignite/pkg/cosmosgen/generate_vuex.go

```diff
@@ -25,25 +25,41 @@ func newVuexGenerator(g *generator) *vuexGenerator {
 }
 
 func (g *generator) updateVueDependencies() error {
-	// Init the path to the "vue" folder inside the app
+	// Init the path to the "vue" and "vuex" folders inside the app
 	vuePath := filepath.Join(g.appPath, "vue")
```

Comment:
```suggestion
	vuePath := filepath.Join(g.appPath, chainconfig.DefaultVuePath)
```

Reply:
it's not a path though...just a flag...i mean technically the default values are the same but it doesnt feel right

Reply:
Yes, a hard coded "vue" path seems off. The user can output the Vue code in a different location using the `--path` flag in the `scaffold vue` command.

Reply:
The right way would be to allow users to indicate the Vue location with a flag when generating Vuex stores I think.

Reply:
> Yes, a hard coded "vue" path seems off. The user can output the Vue code in a different location using the `--path` flag in the `scaffold vue` command.

it's not a path....it's not supposed to change nor does a different path affect this call...
using it to avoid copying the code for composables and hooks because it's the same code...

Could set it up as a  WebAppType enum I guess

Reply:
oh crap,.....scratch that

Reply:
for some reason (not sure why) I thought you were reffering to a different part of the code